src/wallet/rpcwallet.cpp

```diff
@@ -150,6 +150,20 @@ static void addTokenKV(const CTxDestination& address, const CAmount nAmount, Uni
     entry.pushKV("amount", (colorId.type == TokenTypes::NONE ? ValueFromAmount(nAmount) : nAmount ));
 }
 
+static bool checkColorIdParam(const UniValue& param, ColorIdentifier& colorId) {
+    try {
+        const std::vector<unsigned char> vColorId(ParseHex(param.get_str()));
+        if(vColorId.size() != 33) return false;
+        colorId = ColorIdentifier(vColorId);
+        if(colorId.type == TokenTypes::NONE) return false;
+    }
+    catch(...) {
+        colorId.type = TokenTypes::NONE;
```

Comment:
Why this expression need?

Reply:
hmmm...its not needed. I was thinking that this would be useful after return... but its not effective.